SPECS/fdupes/fdupes.spec

```diff
@@ -0,0 +1,226 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
```

Comment:
Please lint the file so that these tags are in the same order as for most other specs we have.